# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: gfx/wr/webrender/src/tile_cache.rs
# Commit: 9fa6f71111d0
# Full Hash: 9fa6f71111d0d77477abc4502b721d8d4ba23604
# Author: Glenn Watson <git@intuitionlibrary.com>
# Date: 2022-02-20 21:58:21
# Regressor Bug: 1749380
# File Overlap Count: 1
# Description:
#   Bug 1749380 - Part 1 - Improve how WR handles bounding rects for off-screen surfaces r=gfx-reviewers
#   
#   This patch introduces a number of subtle but important changes
#   to how we deal with off-screen surfaces. The overall goals are:
#   
# ==============================================================================

diff -r 336a2aa0a29e -r 9fa6f71111d0 gfx/wr/webrender/src/tile_cache.rs
--- a/gfx/wr/webrender/src/tile_cache.rs	Sat Feb 19 23:27:02 2022 +0000
+++ b/gfx/wr/webrender/src/tile_cache.rs	Sun Feb 20 18:56:54 2022 +0000
@@ -7,7 +7,7 @@
 use crate::clip::{ClipChainId, ClipNodeKind, ClipStore, ClipInstance};
 use crate::frame_builder::FrameBuilderConfig;
 use crate::internal_types::{FastHashMap, FastHashSet};
-use crate::picture::{PrimitiveList, PictureCompositeMode, PictureOptions, PicturePrimitive, SliceId};
+use crate::picture::{PrimitiveList, PictureCompositeMode, PicturePrimitive, SliceId};
 use crate::picture::{Picture3DContext, TileCacheParams, TileOffset};
 use crate::prim_store::{PrimitiveInstance, PrimitiveStore, PictureIndex};
 use crate::scene_building::SliceFlags;
@@ -629,7 +629,6 @@
         PrimitiveFlags::IS_BACKFACE_VISIBLE,
         prim_list,
         scroll_root,
-        PictureOptions::default(),
     ));
 
     PictureIndex(pic_index)